tests/Model/NoDriverManagerTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Model;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\MediaBundle\Model\NoDriverManager;
+
+class NoDriverManagerTest extends TestCase
+{
+    /**
+     * @dataProvider providerMethods
+     */
+    public function testException($method, array $arguments)
+    {
+        $this->expectException('Sonata\MediaBundle\Exception\NoDriverException');
+
+        $manager = new NoDriverManager();
```

Comment:
this var is not needed, can you just move `new NoDriverManager()` to method?

Reply:
Fixed =)